independent-publisher-connectors/Influenza and Covid-19/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#f7f7f7",
```

Comment:
@kelcho-spense The default iconBrandColor for any independent connector is `"#da3b01"`. Please follow the [templates](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/Independent%20Publisher).

Reply:
I have rectified the error.